drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/event.rb

```diff
@@ -17,7 +17,9 @@ class Event < GrdaWarehouse::Hud::Base
     def self.involved_warehouse_scope(data_source_id:, project_ids:, date_range:)
       return none unless project_ids.present?
 
-      warehouse_class.joins(enrollment: :project).
+      warehouse_class.
+        where(synthetic: false).
```

Comment:
Let's make an `importable` scope. (in Event that would be `where(synthetic: false)`)  In ProjectCoC etc, `where(manual_entry: false)`